### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leoervik**
Seeing as we specify `Your VCS history, with hopefully more than 1 commit` we should also specify that we want the candidate to deliver the project in a repository. I have some suggestionst:
- [ ] Create a public repo with the code and link to it
- [ ] Create a private repo and ask who should be invited to review
- [ ] Key people from Shortcut fork this repo and create a private repository and invite the candidate as a collaborator

### CODE REVIEW

**Comment by jnutting** [comment]
This makes sense to me!

**Comment by anthomm** [comment]
I agree, while not a big problem, the shy candidates that refuse to leave their repos open make it hard for us to track the time spent as well as other commit history goodies.

**Review by anthomm — APPROVED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ Please include README file to your project. Hopefully it will contain following
 - Add details about trade-offs you have made
 - Add details about things and features you have skipped, or you would like to change if you have additional time for this project.
 
+## Handing over for review
```
Comment (anthomm) [inline README.md:39]: Should we explain WHY we want this done?

Comment (leoervik) [reply README.md:39]: We can ask the candidates if there was any confusion during the process when we have the interview :)

**Pull request merged**